### DISCUSSION TITLE
Reset etcd configuration

### DISCUSSION THREAD

**Discussion by it-naturavelo** [minio/minio#18941] [Q&A]
Hello,
I made a mistake when setting the etcd endpoint on the web admin. now i can't login anymore; i tried to update the setting with mc admin but the access is denied.
minio was deployed with docker
How can i remove or change the etcd endpoint config ?

**Comment by sveinpj** [comment]
Dont know if this is the best resurrection method, but I got it back by:
- Stop the docker or scale deployment if in k8s
- Find the root folder of the store, and navigate to .minio.sys/config/config.json folder
- mv/rename the  xl.meta to xl.meta_bak for backup
- Find a previous xl.meta file in folder ../history/{folderID}.kv/xl.meta and copy it to the .minio.sys/config folder
- restart docker/scale up deployment
Best of luck 👍
